ui/app/AppLayouts/Chat/components/PublicChatPopup.qml

```diff
@@ -77,24 +78,11 @@ ModalPopup {
         }
     }
 
-    footer: Button {
-        width: 44
-        height: 44
+    footer: StatusButton {
         anchors.bottom: parent.bottom
         anchors.right: parent.right
-        background: Rectangle {
-            color: "transparent"
-        }
-        SVGImage {
-            source: channelName.text == "" ? "../../../img/arrow-button-inactive.svg" : "../../../img/arrow-btn-active.svg"
-            width: 50
-            height: 50
-        }
-        MouseArea {
-            id: btnMAJoinChat
-            cursorShape: Qt.PointingHandCursor
-            anchors.fill: parent
-            onClicked : doJoin()
-        }
+        onClicked : doJoin()
+        //% "Start chat"
+        text: qsTrId("start-chat")
```

Comment:
Just a reminder that you can use `qsTr` directly when developing instead if `qsTrId` since it's way easier that way.
Anyway, this is good too, you saved the script some work haha